### PR TITLE
*: Use go.uber.org/zap.Logger everywhere

### PR DESCRIPTION
**Pull request by cthulhu-rider**

### CODE REVIEW

File: cmd/neofs-node/config.go [hunk 1]
```diff
@@ -394,7 +395,6 @@ func (s shared) resetCaches() {
 // dynamicConfiguration stores parameters of the
 // components that supports runtime reconfigurations.
 type dynamicConfiguration struct {
```
Comment (roman-khimov) [inline cmd/neofs-node/config.go:397]: Not needed now?

File: cmd/neofs-node/config.go [hunk 2]
```diff
@@ -989,15 +966,6 @@ func (c *cfg) configWatcher(ctx context.Context) {
 				continue
 			}
 
-			for _, component := range components {
-				err = component.cfg.Reload()
```
Comment (roman-khimov) [inline cmd/neofs-node/config.go:993]: I'm not sure what components are affected by this change. Are we gonna be able to reconfigure log level the way it was possible before?

File: cmd/neofs-node/config.go (hunk 1 above)
Comment (cthulhu-rider) [reply cmd/neofs-node/config.go]: dropped

File: cmd/neofs-node/config.go (hunk 2 above)
Comment (cthulhu-rider) [reply cmd/neofs-node/config.go:993]: yep, `configWatcher` still reconfigures logger but in a different way (decoding `zap.AtomicLevel`)

**Pull request merged**